src/toast/ops/cadence_map.py

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) 2021 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+
+import h5py
+import os
+from time import time
+import warnings
+
+from astropy import units as u
+import numpy as np
+import traitlets
+
+from ..mpi import MPI, MPI_Comm, use_mpi, Comm
+
+from .operator import Operator
+from .. import qarray as qa
+from ..data import Data
+from ..timing import function_timer
+from ..traits import trait_docs, Int, Unicode, Bool, Dict, Quantity, Instance
+from ..utils import Logger, Environment, Timer, GlobalTimers, dtype_to_aligned
+from ..observation import default_names as obs_names
+
+
+def to_JD(t):
+    # Unix time stamp to Julian date
+    # (days since -4712-01-01 12:00:00 UTC)
+    return t / 86400.0 + 2440587.5
+
+
+def to_MJD(t):
+    # Convert Unix time stamp to modified Julian date
+    # (days since 1858-11-17 00:00:00 UTC)
+    return to_JD(t) - 2400000.5
```

Comment:
Are these functions useful for multiple operators?  Should they go in `toast.utils` or `toast.coordinates`?  We can leave them here for now, but if we find ourselves defining them in multiple places then we should move them someplace they can be imported by multiple bits of code.

Reply:
Excellent point. I moved them to `toast.coordinates` and will use them in the other PRs under review.